honeybee_grasshopper_core/src/HB Aperture.py

```diff
@@ -0,0 +1,82 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Create Honeybee Face
+-
+
+    Args:
+        _geo: Rhino Brep geometry.
+        _name_: A name for the Aperture. If the name is not provided a random
+            name will be assigned
+        operable_: Boolean to note whether the Aperture can be opened for ventilation.
```

Comment:
Coming back here after checking the Grasshopper file! I think we need to clarify how this input affects energy and daylight models.

Reply:
Agreed but the truth is that I am not so sure exactly how it will affect the energy simulation until I implement natural ventilation in honeybee-energy. Whatever I put now is going to change, (probably before the first release). Could we just agree that I'll change this once I implement Natural Ventilation?

Reply:
We have an issue for this now ans we will address it when the time comes.